app/ante/ante.go

```diff
@@ -0,0 +1,51 @@
+package ante
```

Comment:
I need more details on the ante changes. It looks like it is quite different from what we are expecting from cosmos/evm guide